_translator-files/po/de/Installation-for-iOS.po

```diff
@@ -3,149 +3,150 @@
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:21
 msgid "If you don't pay for an Apple developer account, Jamulus will only run for at most 7 days before you need to re-install it."
-msgstr ""
+msgstr "Wenn du nicht für ein Apple-Entwicklerkonto bezahlst, läuft Jamulus höchstens 7 Tage lang, bevor du es erneut installieren musst."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:22
 msgid "Jamulus on iOS works quite well on new devices if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. To connect these to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered) USB-hub."
-msgstr ""
+msgstr "Jamulus auf iOS funktioniert auf neuen Geräten recht gut, wenn alles richtig eingestellt ist. Wir empfehlen dir jedoch dringend, ein Audio-Interface und einen USB-Ethernet-Adapter zu verwenden. Um diese mit deinem iOS-Gerät zu verbinden, solltest du etwas wie den [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) und einen (mit Strom versorgten) USB-Hub verwenden."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:26
 msgid "## Install for iOS (for experienced users only)  **Note**: We provide an unsigned .ipa file which you need to sign and then install on your device. If you are not Jailbroken (and have AppSync installed), you might need additional software to install Jamulus. There are multiple ways to do that, but this guide only explains the most standard approach. Please note that we assume you already have a (free) Apple developer account. If you don't have one, create an Apple ID and [Sign up for a developer account](https://developer.apple.com/membercenter). We recommend to create one without two factor authentification and unrelated to your potential personal account. You will need to enter the password in non-Apple software."
-msgstr ""
+msgstr "## Installation für iOS (nur für erfahrene Benutzer) **Hinweis**: Wir stellen eine unsignierte .ipa-Datei zur Verfügung, die du signieren und dann auf deinem Gerät installieren musst. Wenn dein Gerät nicht Jailbroken ist (und AppSync installiert hast), benötigst du möglicherweise zusätzliche Software, um Jamulus zu installieren. Dafür gibt es mehrere Möglichkeiten, aber in dieser Anleitung wird nur die gängigste Methode beschrieben. Bitte beachte, dass wir davon ausgehen, dass du bereits ein (kostenloses) Apple-Entwicklerkonto hast. Wenn du noch keines hast, erstelle eine neue Apple ID und [registriere dich für ein Entwicklerkonto](https://developer.apple.com/membercenter). Wir empfehlen dir, ein Konto ohne Zwei-Faktor-Authentifizierung zu erstellen, das nicht mit deinem persönlichen Konto verknüpft ist. Du musst das Passwort in die Nicht-Apple-Software eingeben."
```

Comment:
Isn't a new line missing here between the headline and the "note" 

@ignotus666 Is that valid po syntax?